extensions/wikia/BannerNotifications/templates/BannerNotifications.mustache

```diff
@@ -0,0 +1,6 @@
+<div class="banner-notification">
+	<button class="close wikia-chiclet-button">
+		<img src="{{imageSource}}">
```

Comment:
Is it possible to render this image using CSS?
This way we could limit 1 request for assets and make JS code cleaner.